src/applications/gi/components/vet-tec/VetTecProgramSearchResult.jsx

```diff
@@ -60,13 +60,15 @@ class VetTecProgramSearchResult extends React.Component {
                 {renderPreferredProviderFlag(this.props.result)}
               </div>
             </div>
-            {(schoolClosing || cautionFlags.length > 0) && (
+            {(schoolClosing || cautionFlags) && (
               <div className="row alert-row">
                 <div className="small-12 columns">
                   {renderSchoolClosingAlert({ schoolClosing })}
-                  {renderCautionAlert({
-                    cautionFlags,
-                  })}
+                  {cautionFlags &&
+                    cautionFlags.length > 0 &&
+                    renderCautionAlert({
+                      cautionFlags,
+                    })}
```

Comment:
For this additional check you could have used [optional chaining](https://dsva.slack.com/archives/CBU0KDSB1/p1567182149038900)